contents/blog/mykonos-hackathon.md

```diff
@@ -0,0 +1,172 @@
+
+
+As PostHog grows as a company, keeping track of everything that’s happening will get harder. Yes, we [write everything down](/newsletter/remote-working) but that creates a lot of reading, and a lot of noise. The solution? An AI-generated briefing, tailored to each individual team member and their interests.
+
+The presidential briefing was built by scraping PRs and issues from GitHub, along with Slack messages, and then training an LLM to understand what’s interesting and important. The bot then produces a pithy briefing that removes the noise and gives people just the information they need. 
+
+While just a proof-of-concept for now, if we were to ship it we’d want to add more data sources and build it using Llama to avoid the need to send any data to external services.
+
+### MykoLogs
+
+<BuiltBy people={["Paul D'Ambra", "Tom Owers", "Ted Kaemming"]} />
+
+MykoLogs is a logging product that integrates with the existing PostHog SDKs, bringing backend logs straight into a shiny new product on PostHog. It's internal-only for now, but could one day be made public. The best part? You can link backend logs to session recordings through the user’s session ID – letting you debug what was happening on the backend during your user's API requests. MykoLogs plays nicely with all other PostHog products, meaning logs and session replay are now BFFs. Debugging has never been this breezy!
+
+### Our own programming language
+
+<BuiltBy people={["Marius Andra", "Anirudh Pillai"]} />
+
+What’s cooler than having your own query language? Having your own general purpose programming language, of course. That’s literally what we built: PostHog’s first ever programming language, Hog. (Earlier versions of this name included Hög and Höge, but turns out we're anti-umlaut.)
+
+Things moved fast the week after Mykonos: we split up the existing product analytics team and built a new team to productize Hog. We plan to use Hog to build our CDP and messaging products, and might even pivot the entire company around it. It was just another PostHog hackathon, no big deal. Stay tuned for the aftermath.
+
+### RealTimeHog 3000
+
+<BuiltBy people={["James Greenhill", "Zach Waterfield", "Michael Matloka", "Brett Hoerner"]} />
+
+Seeing people using your product live boosts dopamine levels. Probably. PostHog does that – but right now we keep you waiting a bit. Before a PostHog event is available for querying, it goes through our ingestion pipeline, where users are identified and the events themselves enriched. The process takes seconds, rarely a minute – a delay imperceptible in analytical queries, but a dopamine decrease in the live view.
+
+Introducing RealTimeHog 3000, a [livestream](https://github.com/PostHog/livestream) service powering our first truly real-time view of user activity. The process is simple: 
+
+1. Consume the raw events from the same Kafka topic as our ingestion service. 
+2. Stream them to you ASAP using server-sent events. 
```

Comment:
```suggestion
Introducing RealTimeHog 3000, a [livestream](https://github.com/PostHog/livestream) service powering our first truly real-time view of user activity. The process is simple: 

1. Consume the raw events from the same Kafka topic as our ingestion service. 
2. Stream them to you ASAP using server-sent events. 

[GIF / VIDEO CLIP HERE]
```

This definitely needs a video clip of the feature. 15 to 20 seconds of the stream of live events from the demo would be enough.